packages/grafana-ui/src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -51,11 +52,18 @@ const getStyles = (_theme: GrafanaTheme2, delay: number, duration: number) => {
       transform: 'translateX(-100%)',
       animationName: animation,
       // an initial delay to prevent the loader from showing if the response is faster than the delay
-      animationDelay: `${delay}ms`,
-      animationDuration: `${duration}ms`,
       animationTimingFunction: 'linear',
       animationIterationCount: 'infinite',
       willChange: 'transform',
+      ...handleReducedMotion(
+        {
+          animationDelay: `${delay}ms`,
```

Comment:
i don't think you want to put the delay in here - the delay should always be there